app/Http/Controllers/SettingsController.php

```diff
@@ -46,22 +47,22 @@ public function storeAppName(Request $request)
     }
 
 
-    public function storeAppLogo(Request $request)
-    {
-        $request->validate([
-            'app_logo' => 'required|image|mimes:jpg,png,jpeg,gif,svg|max:2048',
-            'label' => 'required'
-        ]);
-        $path = 'storage/' . Storage::disk('public')->putFileAs(
-            'logos',
-                $request->file('app_logo'),
-                now()->format('d-m-y-H-i-s').'.'.$request->file('app_logo')->extension());
-        Settings::updateOrCreate(
-            ['slug' => $request->label],
-            ['data' => ['path' => $path]]
-        );
-        return back();
-    }
+  public function storeAppLogo(Request $request)
```

Comment:
2-space indents are indeed much easier to read for narrow-screens (and in github's UI)

Reply:
true but also PSR2